### ISSUE TITLE
About training sets

### ISSUE THREAD

**Issue by yuanpeikai** [migalkin/StarE#10]
Hi,I found that your training set has validation set data.Could you tell me if a validation set is used for training？

Problem code：
ev_vl_data = {'index': combine(train_data, valid_data), 'eval': combine(test_data)}
ev_tr_data = {'index': combine(valid_data, test_data), 'eval': combine(train_data)}
tr_data = {'train': combine(train_data, valid_data), 'valid': ev_vl_data['eval']}

You use tr_data  for training.

**Comment by migalkin** [comment]
Hi, yes we merge train + validation when training the final model with evaluating on the test set, eg. when `USE_TEST True`. When doing hyperparam optimization on the validation set, only the training set is used as seen in the code:

https://github.com/migalkin/StarE/blob/f40a5ee082d61851477e9870c21e991c7d91deb3/run.py#L298-L305

**Comment by yuanpeikai** [comment]
Thanks for your reply.
